packages/lib/services/KeymapService.ts

```diff
@@ -112,6 +113,7 @@ const defaultKeymapItems = {
 		{ accelerator: 'Ctrl+Alt+3', command: 'switchProfile3' },
 		{ accelerator: 'Ctrl+Alt+N', command: 'openNoteInNewWindow' },
 		{ accelerator: 'Ctrl+M', command: 'toggleTabMovesFocus' },
+		{ accelerator: 'Alt+F6', command: 'focusToolbar' },
```

Comment:
Could you please put this shortcut below all the other `focusXxx` shortcuts?

Also what was the reason for choosing a completely different keyboard shortcut for this? All the other ones are "Ctrl+Shift+Something"